src/src/luaParams.cpp

```diff
@@ -68,4 +68,44 @@ struct tagLuaItem_command luaWebUpdate = {
     LUA_COMMAND_SIZE(luaWebUpdate)
 };
 #endif
+
+//----------------------------VTX ADMINISTRATOR------------------
+
+struct tagLuaItem_textSelection luaVtxBand = {
+    {0,(uint8_t)CRSF_TEXT_SELECTION},//id,type
+    "Band",
+    "Off;A;B;E;F;R;L",
+    {0,0,6},//value,min,max
+    emptySpace,
+    LUA_TEXTSELECTION_SIZE(luaVtxBand)
+};
+
+struct tagLuaItem_uint8 luaVtxChannel = {
+    {0,(uint8_t)CRSF_UINT8},//id,type
+    "Channel",
+    {0,0,7},//value,min,max
+    emptySpace,
+    LUA_UINT8_SIZE(luaVtxChannel)
+};
+
+struct tagLuaItem_textSelection luaVtxPwr = {
+    {0,(uint8_t)CRSF_TEXT_SELECTION},//id,type
+    "VTX Pwr",
+    "10;25;50;100;250;500;1000;2000",
```

Comment:
These also need to be just numbered NC;1;2;3;4;5;6;7;8 again as we don't really know what power levels the user has configured in the VTX tables or indeed what the VTX uses.

Reply:
Agreed, Ive used "-" in place of NC. 0 power index is ignored by BF and the power level is unchanged.